packages/react-lowcode/src/codegen/generation/generators/list/grommet-dt-generator.ts

```diff
@@ -60,8 +153,8 @@ export default class GrommetDataTableGenerator implements TableGenerator
     private createColumns(columnsIdentifier: ts.Identifier):ts.VariableDeclarationList {
         let propertiesColumnDefinitions = Array<ts.ObjectLiteralExpression>();
 
-        getProperties(this._entity).forEach(property => {
-            propertiesColumnDefinitions.push(this.createColumnDefinition(property));
+        getProperties(this._entity!).forEach(property => {
+            propertiesColumnDefinitions.push(this.createColumnDefinition(property, this._context.formatter??Formatter.None));
```

Comment:
you can move the default value to the initialization/constructor probably...

Reply:
fixed